python/mxnet/symbol.py

```diff
@@ -483,7 +555,23 @@ def list_outputs(self):
         return [py_str(sarr[i]) for i in range(size.value)]
 
     def list_auxiliary_states(self):
-        """List all auxiliary states in the symbol.
+        """Lists all the auxiliary states in the symbol.
+
+        Example usage:
+        ----------
+        >>> a = mx.sym.var('a')
+        >>> b = mx.sym.var('b')
+        >>> c = a + b
+        >>> c.list_auxiliary_states()
+        []
+
+        Auxiliary states list with batch norm.
```

Comment:
"Auxiliary states list with batch norm." -> "Listing of auxiliary states with batch norm."